app/src/main/java/com/github/libretube/helpers/PlayerHelper.kt

```diff
@@ -289,6 +289,12 @@ object PlayerHelper {
             "18"
         ).toFloat()
 
+    val fullscreenCaptionsTextSize: Float
+        get() = PreferenceHelper.getString(
+            PreferenceKeys.CAPTIONS_SIZE,
+            "27"
+        ).toFloat()
```

Comment:
You didn't add a preference for that yet, so the user can't change it. And  I think we should just make the font size 1.5 times bigger instead of adding a new preference.

Reply:
Sure.